docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -797,3 +799,30 @@ By default, a request would need to satisfy all authentication requirements in o
 ```yaml
 nginx.ingress.kubernetes.io/satisfy: "any"
 ```
+
+### Mirror
+
+Enables a request to be mirrored to a mirror backend. Responses by mirror backends are ignored. This feature is useful, to see how requests will react in "test" backends.
+
+You can mirror a request to the `/mirror` path on your ingress, by applying the below:
+
+```yaml
+nginx.ingress.kubernetes.io/mirror-uri: "/mirror"
+```
+
+The mirror path can be defined as a separate ingress resource:
+
+```
+location = /mirror {
+    internal;
+    proxy_pass http://test_backend;
+}
+```
+
+By default the request-body is sent to the mirror backend, but can be turned off by applying:
+
+```yaml
+nginx.ingress.kubernetes.io/mirror-request-body: "off"
+```
+
+For more information on the mirror module see https://nginx.org/en/docs/http/ngx_http_mirror_module.html
```

Comment:
Please add a note mentioning the side effects of using this feature against a slow backend
https://alex.dzyoba.com/blog/nginx-mirror/

Reply:
👍